golem-component-service-base/tests/db/migration/postgres/001__init.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE components
+(
+    namespace           text    NOT NULL,
+    component_id        uuid    NOT NULL PRIMARY KEY,
+    name                text    NOT NULL,
+    UNIQUE (namespace, name)
+);
+
+CREATE INDEX components_namespace_id_idx ON components (namespace, component_id);
```

Comment:
This index is useless. For search by `namespace` there is already `UNIQUE (namespace, name)`. For search by ID - PK